scripts/label_all_tokens.py

```diff
@@ -86,8 +91,13 @@ def main():
         labelled_token_ids_dict[token_id] = labels[0][0]
 
     # Save the labelled tokens to a file
-    filename = "labelled_token_ids_dict.pkl"
-    filepath = STATIC_ASSETS_DIR.joinpath(filename)
+    if os.path.split(args.output)[0] == "":
+        filepath = STATIC_ASSETS_DIR.joinpath(args.output)
+        print(f"Outputting file {args.output} to path {filepath}")
+    else:
+        filepath = os.path.expandvars(args.output)
```

Comment:
why?

Reply:
e.g. `$HOME`

Reply:
Oh, sorry, think I misinterpreted this - you're asking "why" about the whole block, in which case the answer is "because that's the straightforward reading of the issue that I wrote". I should have just made the issue "use the path the user gives without any reference to the static dir" - so that's on me.

Reply:
No, I was asking just about expandvars - I haven't seen this before